catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -235,6 +282,50 @@ public Metacard transform(InputStream input, String id)
         }
     }
 
+    private String getDatatype(String mimeType) {
+        if (mimeType == null) {
+            return null;
+        }
+
+        LOGGER.debug("Attempting to map {}", mimeType);
+        Optional<Map.Entry<com.google.common.net.MediaType, String>> returnType =
+                SPECIFIC_MIME_TYPE_DATA_TYPE_MAP.entrySet()
+                        .stream()
+                        .filter(mediaTypeStringEntry -> {
+                            com.google.common.net.MediaType mediaType =
+                                    com.google.common.net.MediaType.parse(mimeType);
+                            return mediaType.is(mediaTypeStringEntry.getKey());
+                        })
+                        .findFirst();
+
+
+        if (!returnType.isPresent()) {
+            Optional<Map.Entry<com.google.common.net.MediaType, String>> fallback = FALLBACK_MIME_TYPE_DATA_TYPE_MAP.entrySet()
+                    .stream()
+                    .filter(mediaTypeStringEntry -> {
+                        com.google.common.net.MediaType mediaType =
+                                com.google.common.net.MediaType.parse(mimeType);
+                        return mediaType.is(mediaTypeStringEntry.getKey());
+                    }).findFirst();
+
+            if(fallback == null || !fallback.isPresent()) {
+                return OVERALL_FALLBACK_DATA_TYPE;
+            }
+
+            return FALLBACK_MIME_TYPE_DATA_TYPE_MAP.entrySet()
```

Comment:
I don't understand why you're reprocessing the fallback stream here. If `fallback` is present, you should simply be able to return `fallback.get().getValue()`, right?